src/app/components/Embeds/OEmbed/index.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render } from '../../react-testing-library-with-providers';
+
+import OEmbedLoader from '.';
+
+const riddleBlock = {
```

Comment:
Per my comment above - should we have 1 fixture file for this, and import it into all tests/stories?